src/main/java/com/internship/frejaeidjmeterplugin/jmeter/visualizer/gui/FrejaGraphVisualizerGui.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.internship.frejaeidjmeterplugin.jmeter.visualizer.gui;
+
+import java.awt.BorderLayout;
+import java.awt.Component;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.StringTokenizer;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.swing.JTabbedPane;
+import org.apache.jmeter.samplers.SampleResult;
+import org.apache.jmeter.visualizers.gui.AbstractVisualizer;
+import org.knowm.xchart.QuickChart;
+import org.knowm.xchart.XChartPanel;
+import org.knowm.xchart.XYChart;
+
+/**
+ *
+ * @author User
+ */
+public class FrejaGraphVisualizerGui extends AbstractVisualizer {
+    List<Double> latency;
+    List<Double> successCount;
+    List<Double> failCount;
+    List<Double> count;
+    GraphPanel authResults;
+    GraphPanel signResults;
+    GraphPanel openSecureResults;
+    HashMap<String, GraphPanel> panelList;
+    private static final String RESPONSE_CODE = "FAILED";
+
+    public FrejaGraphVisualizerGui() {
+        super();
+        panelList = new HashMap<>();
+        setLayout(new BorderLayout(0, 5));
+        setBorder(makeBorder());
+        add(makeTitlePanel(), BorderLayout.NORTH);
+        authResults = new GraphPanel("Authentication Sampler Results", "Number" ,"Count");
```

Comment:
move this to separate private method called createFrejaEidResultsPanels() or something like this.